client/my-sites/people/delete-user/index.jsx

```diff
@@ -79,18 +79,12 @@ class DeleteUser extends React.Component {
 		updateObj[ name ] = value;
 
 		this.setState( updateObj );
-		analytics.ga.recordEvent( 'People', 'Selected Delete User Assignment', 'Assign', value );
+		this.props.recordGoogleEvent( 'People', 'Selected Delete User Assignment', 'Assign', value );
 	};
 
-	setReassignLabel = label => {
-		this.reassignLabel = label;
-	};
+	setReassignLabel = label => ( this.reassignLabel = label );
```

Comment:
while you're at it you might check out the preferred `createRef()` syntax
https://reactjs.org/docs/react-api.html#reactcreateref

Reply:
Thanks, I think that notion, or at least a ghost of it passed through my mind. More elegant than a callback ref for sure. 

Maybe for a second pass later?